### DISCUSSION TITLE
Expo Push Notifications Not Working | Push notifications tool

### DISCUSSION THREAD

**Discussion by dananjaya6005** [expo/expo#30912] [Archived: Questions] [answered]
I'm experiencing an issue with Expo Push Notifications. Everything was working fine yesterday ( 2024.08.08), but today(2024.08.09) , push notifications are not being delivered for both my existing production app and my development client build.

I have verified that all credentials are correctly set up, and I have also tested notifications using the Expo Push Notifications tool, but have not received any results.

**Comment by osoftz** [comment]
We are also experiencing the same issue today. Yesterday it worked properly and it is not working. Can anyone from the expo team explain to us what is happening?

**Comment by veeramani042** [comment]
Yeah I'm also facing the same issue. from server side it gives ok response but notification not received, Any solutions?

**Comment by hochonin93** [comment]
I experienced the same issue. It was working fine yesterday.

**Reply by dananjaya6005** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/41192363-b3c6-4db7-afc4-388a5beba726)

**Reply by veeramani042** [reply]
@dananjaya6005 Thanks for the info

**Comment by dananjaya6005** [comment]
![356662147-41192363-b3c6-4db7-afc4-388a5beba726](https://github.com/user-attachments/assets/59f3c3c5-d28f-4ee0-a8f8-580f8c2cf93a)
